### PR TITLE
Dev/disable quilt

### PR DESCRIPTION
**Pull request by duplexsystem**
Disabled quilt because it causes our CI to run out of memory. If you're dead set on native quilt support, you can manually re-enable it, but it will be an unsupported build. It's ok to just disable quilt entirely because the fabric impl works on quilt and for some reason the quilt impl needs to be more hacky (forge flashbacks) than fabric despite both working on quilt.

### CODE REVIEW

File: platforms/mixin-common/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # mixin-common
 
-This project contains mixins shared between Forge, Fabric and Quilt, as
+This project contains mixins shared between Forge, Fabric and formerly Quilt, as
```
Comment (justaureus) [inline platforms/mixin-common/README.md:3]: Change this line to "This project contains mixins shared between Forge and Fabric, as"?

File: platforms/mixin-lifecycle/README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # mixin-lifecycle
 
-This project contains lifecycle mixins shared between Fabric and Quilt.
\ No newline at end of file
+This project contains lifecycle mixins shared between Fabric and formerly Quilt.
```
Comment (justaureus) [inline platforms/mixin-lifecycle/README.md:3]: Similarly to above change this line to "This project contains lifecycle mixins for Fabric."  Seems redundant to mention Quilt if it's being removed.

**Review by justaureus — COMMENTED** [review]
Looks good other than those two minor readme notes.

File: buildSrc/src/main/kotlin/Versions.kt [hunk 3]
```diff
@@ -57,13 +57,13 @@ object Versions {
...
     }
-    
-    object Sponge {
-        const val sponge = "9.0.0-SNAPSHOT"
-        const val mixin = "0.8.2"
-        const val minecraft = "1.17.1"
-    }
-    
+//
+//    object Sponge {
```
Comment (solonovamax) [inline buildSrc/src/main/kotlin/Versions.kt:61]: Why is sponge commented out in a PR relating to quilt?

**Review by solonovamax — APPROVED** [review]
LGTM

File: buildSrc/src/main/kotlin/Versions.kt (hunk 3 above)
Comment (duplexsystem) [reply buildSrc/src/main/kotlin/Versions.kt:61]: just minor clean up

**Pull request merged**